samples/python/tutorial_code/ShapeDescriptors/point_polygon_test/pointPolygonTest_demo.py

```diff
@@ -46,6 +46,7 @@
             drawing[i,j,1] = 255
             drawing[i,j,2] = 255
 
+cv.circle(drawing,maxDistPt, np.int(maxVal),(255,255,255), 1, cv.LINE_8, 0)
```

Comment:
Respect the current coding style of the file and add space after each comma `,` (except may be tuple `(255,255,255)`).

> np.int(maxVal)

Just `int(maxVal)` should work too.

Reply:
great suggestion!I will modifiy the code as soom as possible